src/mail.sh

```diff
@@ -0,0 +1,170 @@
+include "$KW_LIB_DIR/kwlib.sh"
+include "$KW_LIB_DIR/kwio.sh"
+include "$KW_LIB_DIR/kw_config_loader.sh"
+
+# Hash containing user options
+declare -gA options_values
+
+function kw_mail()
+{
+  # say "enter"
+  parse_mail_options "$@"
+  if [[ "$?" -gt 0 ]]; then
+    complain "${options_values['ERROR']}"
+    return 22 # EINVAL
+  fi
+
+  if [[ "${options_values['SETUP']}" == 1 ]]; then
+    mail_setup
+    exit
+  fi
+
+  cmd="git send-email $opts --to=$recipients -1"
+
+  # TODO: eval "$cmd"
+  echo "$cmd"
+
+  return 0
+}
+
+function mail_setup()
+{
+  local global=''
+
+  if [[ ! -d "$PWD/.git" ]] && [[ $(ask_yN 'Not in a git repository, apply changes globally?') =~ 1 ]]; then
+    global='--global'
+  fi
```

Comment:
Maybe we should have a flag named `--global` in the `mail` option. If the user does not provide it, we assume local change.